Documentation/configure-kubectl.md

```diff
@@ -1,63 +1,42 @@
 # Setting up kubectl
 
-`kubectl` is a command-line program for interacting with the Kubernetes API. The following steps should be done from a local workstation to configure `kubectl` to work with a new cluster.
+Use kubectl to interact with the Kubernetes API and with the cluster’s shared state. Download kubectl from the Kubernetes release artifact site with the curl tool.
 
-To quickly launch a cluster, follow these guides for [AWS][kube-aws], [Vagrant][vagrant-multi] or [full step-by-step][manual] instructions.
+Use curl to fetch the Linux kubectl binary:
 
-[kube-aws]: https://github.com/coreos/kube-aws/blob/master/README.md
-[vagrant-multi]: kubernetes-on-vagrant-single.md
-[manual]: getting-started.md
-
-## Download the kubectl Executable
-
-Download `kubectl` from the Kubernetes release artifact site with the `curl` tool.
-
-The linux `kubectl` binary can be fetched with a command like:
-
-```sh
-$ curl -O https://storage.googleapis.com/kubernetes-release/release/v1.5.4/bin/linux/amd64/kubectl
+```
+$ curl -O https://storage.googleapis.com/kubernetes-release/release/v1.6.1/bin/linux/amd64/kubectl
 ```
 
-On an OS X workstation, replace `linux` in the URL above with `darwin`:
+Or, to fetch the macOS binary:
 
-```sh
-$ curl -O https://storage.googleapis.com/kubernetes-release/release/v1.5.4/bin/darwin/amd64/kubectl
+```
+$ curl -O https://storage.googleapis.com/kubernetes-release/release/v1.6.1/bin/darwin/amd64/kubectl
 ```
 
-After downloading the binary, ensure it is executable and move it into your `PATH`:
+After downloading the binary, ensure it is executable and move it into your PATH:
 
-```sh
+```
 $ chmod +x kubectl
-$ mv kubectl /usr/local/bin/kubectl
+$ sudo mv kubectl /usr/local/bin/kubectl
 ```
 
-## Configure kubectl
+Terraform generated a set of files, including a `kubeconfig`, which specifies the credentials for your cluster. Once logged in to the Tectonic Console, other users can download their own pre-generated kubeconfigs.
 
-Configure `kubectl` to connect to the target cluster using the following commands, replacing several values as indicated:
+As the first admin, you have a special “root” kubeconfig. Configure `kubectl` to use this file:
 
-* Replace `${MASTER_HOST}` with the master node address or name used in previous steps
-* Replace `${CA_CERT}` with the absolute path to the `ca.pem` created in previous steps
-* Replace `${ADMIN_KEY}` with the absolute path to the `admin-key.pem` created in previous steps
-* Replace `${ADMIN_CERT}` with the absolute path to the `admin.pem` created in previous steps
+```
+$ export KUBECONFIG=/path/to/installer/generated/auth/kubeconfig
```

Comment:
maybe expand this section to say that you can set this ephemerally for the terminal window/session, or set it permanently by moving it to `~/kube/config`. Maybe also mention how you could best manage two of these?

Reply:
I think just offer one happy path, to get the user through the tutorial. Going into detail on how to configure both a terminal-specific path, and a permanently set location may be too much info for this quick start guide. but I could be wrong.

Reply:
This isn't currently linked from any tutorials, right? Or are you saying that that the plan is to do that?

Reply:
no - these pages *are* a tutorial. I don't think they should go into detail on multiple ways to manage kubeconfig.

Reply:
Why not? This is the second hit for "install kubectl", it seems reasonable that folks will want to know how to best manage their kubeconfigs. We know that folks frequently run multiple clusters.

Reply:
It can be at the bottom or somehow indicated that you're done, but continue if you want to learn how to best manage your kubeconfig.